### ISSUE TITLE
Laravel

### ISSUE THREAD

**Issue by ShapesGraphicStudio** [rollincode/RCPaiementCMCICBundle#15]
Bonjour,
Désolé pour ce post qui n'est qu'une question, pas vraiment une issue.
Est-ce que ce bundle pourrait fonctionner avec Laravel ?
Bien cordialement,
David

**Comment by matthv** [comment]
Bonjour,
Il vaut mieux que tu effectues un "fork" de ce bundle pour l'adapter sur Laravel.
Le bundle utilise un Form : PaiementType qui hérite de la classe Symfony\Component\Form\AbstractType qui ne sera pas compatible avec Laravel.

Et je ne parle pas de la dépendance dans composer de Symfony, après toute la logique devrait être bonne à reprendre.

cc @nicovak

**Comment by ShapesGraphicStudio** [comment]
Merci pour la réponse @matthv 
C'est bien ce que je pensais vu les points communs et différences entre les 2 frameworks.
Je ne trouve étonnamment aucun bundle déjà développé pour Laravel.

**Comment by matthv** [comment]
Pas de soucis :)
Oui effectivement, je viens de regarder il n'y a pas de solution en package laravel actuellement.
Après, notamment, en France les devs php utilisent plus symfony que laravel et CMCIC, c'est français et ça n'aide pas ^^.
Si tu as du temps, faire un package laravel n'est pas très compliqué [documentation-package-laravel](https://laravel.com/docs/5.8/packages)
Bon courage pour ton paiement.

Signé un autre dev Laravel ^^

**Comment by ShapesGraphicStudio** [comment]
Oui, on peut dire que la combinaison Laravel + CMCIC est plutôt spécifique ! ;-) 
Je n'ai pas encore fait de packages Laravel, il faut que j'y jette un œil.
Merci encore !
